runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -421,6 +444,12 @@ void J9::RecognizedCallTransformer::transform(TR::TreeTop* treetop)
       case TR::java_lang_StringUTF16_toBytes:
          process_java_lang_StringUTF16_toBytes(treetop, node);
          break;
+      case TR::java_lang_StrictMath_sqrt:
+         process_java_lang_StrictMath_sqrt(treetop, node);
+         break;
+      case TR::java_lang_Math_sqrt:
+         processIntrinsicFunction(treetop, node, TR::dsqrt);
```

Comment:
Why do we need to distinguish between these two? Shouldn't they be the same?

Reply:
The reason we distinguish them is that `TR::java_lang_Math_sqrt` is not a native function. There won't be a `jclass` child with it. And `TR::java_lang_StrictMath_sqrt` is a native function with two children. So I want to process the two-children case separately.